cmd/network/verification.go

```diff
@@ -307,6 +317,8 @@ func (e *EgressVerification) generateAWSValidateEgressInput(ctx context.Context,
 	e.log.Info(ctx, "using platform: %s", platform)
 	input.PlatformType = platform
 
+	input.CPUArchitecture = e.cpuArch
```

Comment:
Is there a use-case for this feature at this time? All clusters are x86 right? HCP clusters in the future can be multiarch (mix of x86/arm), how would an SRE decide which architecture to use (does it matter)?

If there's some criteria, I would expect this to be auto-populated from OCM data about the cluster for SRE.

Reply:
Eventually we'll probably determine it automatically from `--cluster-id` like the other fields, but this will allow us to start testing asap against AWS enviroments.